app/src/main/java/com/cliqz/browser/main/TabsManager.java

```diff
@@ -53,8 +49,8 @@ public Builder setForgetMode(boolean value) {
             return this;
         }
 
-        public Builder setOriginTab(TabFragment from) {
-            mFromTab = from;
+        @SuppressWarnings({"unused", "WeakerAccess"})
+        public Builder setOriginTab(TabFragment2 from) {
```

Comment:
Can we remove this if it's not doing anything?

Reply:
I would like to keep it for future